modules/jooby-avaje-inject/src/main/java/io/jooby/avaje/inject/AvajeInjectModule.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Jooby https://jooby.io
+ * Apache License Version 2.0 https://jooby.io/LICENSE.txt
+ * Copyright 2014 Edgar Espina
+ */
+package io.jooby.avaje.inject;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.typesafe.config.Config;
+
+import io.avaje.inject.BeanScope;
+import io.avaje.inject.BeanScopeBuilder;
+import io.jooby.Extension;
+import io.jooby.Jooby;
+
+/**
+ * Avaje Inject module: https://jooby.io/modules/avaje-inject.
+ *
+ * <p>Jooby integrates the {@link io.jooby.ServiceRegistry} into the Avaje DI framework.
+ *
+ * <p>Usage:
+ *
+ * <pre>{@code
+ * {
+ *
+ *
+ *   install(new AvajeInjectModule());
```

Comment:
should be: `install(AvajeInjectModule.of())` or we need to make constructor public